README.md

```diff
@@ -60,6 +60,36 @@ client.trackMetric("custom metric", 3);
 client.trackTrace("trace message");
 ```
 
+### Telemetry Processor
+
+```javascript
+public addTelemetryProcessor(telemetryProcessor: (envelope: ContractsModule.Contracts.Envelope) => boolean)
+```
+
+Adds telemetry processor to the collection. Telemetry processors will be called one by one before telemetry item is pushed for sending and in the order they were added. 
+If one of telemetry processors returns false then telemetry item will not be sent. 
+If one of telemetry processors throws an error then telemetry item will not be sent.
+
+**Example**
+
+Add the below code before you send any telemetry, it will remove stack trace information from any Exception reported by the SDK.

```

Comment:
This code is just an example of how a consumer of the SDK can write a telemetry processor.